window.go

```diff
@@ -14,6 +14,7 @@ import (
 const (
 	EventNameWebContentsEventLogin             = "web.contents.event.login"
 	EventNameWebContentsEventLoginCallback     = "web.contents.event.login.callback"
+	EventNameWebContentsExecuteJavaScript      = "web.contents.execute.javascript"
```

Comment:
I'd rather rename this event and its constant `window.event.web.contents.executed.javascript`

Reply:
This const should be:

`EventNameWindowEventWebContentsExecutedJavaScript = "window.event.web.contents.executed.javascript"`